changelog.d/1206.bugfix

```diff
@@ -0,0 +1 @@
+Create DebugAPI after its dependency ClientPool
```

Comment:
We should mention what bug this fixes, rather than the solution as it's a user facing changelog.

Reply:
Does my new message better fit the purpose of the changelog?